src/components/IconButton/IconButton.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+import IconButton from './IconButton';
+import SectionHeader from '../../storybook/SectionHeader';
+import * as IconButtonStories from './IconButton.stories';
+
+<Meta of={IconButtonStories} />
+
+<SectionHeader title={'IconButton'} />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+
+## Overview
+
+A universal IconButton component that with the help of Button hold the icon inside it and makes
+it clickable and easily accessible.
+
+## Props
+
+<ArgTypes of={IconButton} />
+
+## Usage
+
+<Tip>Upon hovering, it is recommended to always display a label tooltip for an icon button.</Tip>
+
+<SubsectionHeader title="Variants" />
+
+### IconButton Types and Sizes
+
+IconButton with different types `primary, secondary, tertiary` and different shape types `circle, square` <br/>
+There are 2 IconButton sizes, `normal` and `compact`.
+
+<Canvas of={IconButtonStories.IconButtonTypes} />
+
+### Playground
+
+<Canvas of={IconButtonStories.IconButtonPlayground} />
```

Comment:
can we rename this to Playground?

Reply:
fixed here https://github.com/Orfium/orfium-ictinus/pull/727/commits/75b0366b3ffd3b719788f9dda327dbe5aa2a898a